### DISCUSSION TITLE
General question: Does the "User Manager" field do anything?

### DISCUSSION THREAD

**Discussion by MWR-Napavine** [snipe/snipe-it#11242] [General]
Just asking, as I don't see an answer in the documentation; Doe the "Manager" field do anything special on a User? Or is it just a pointer to a different user for database purposes? Does the manager user get any kind of permissions over the managed users?

Also, is there a way to quickly bring up a list of users based on the managed user? My scenario is for a school, I'm thinking of using the Manager field for the "homeroom" teacher of students... that way the homeroom teacher (or more likely me, the Inventory admin) can bring up a list of the students and their assets for an audit.

Thanks for any info!

**Comment by NateLapT** [comment]
I would love if this was used in a near future release! 
When I click email list of checkouts it should cc the Manager of said user and the reply email should be to the Manager not the system admin.

**Reply by qay21** [reply]
As of today, a manager can see all of his·her employees as the default view in the Users tab. His·her employees are, seemingly, users whose responsible is him·her, as well as users whose department's manager is him·her. To see this, the manager must have a role assigned, allowing at least the "View" privilege on Users.

Moreover, if the manager also has the "View" privilege on assets, accessories, licences, etc, he·she will be able to see employees' assigned items.

All of the above might be altered by the "Company" field of each item/user : a non-admin cannot view items that does not belong to his·her company.

So the manager of a user/department actually HAVE some kind of features nowadays. It is not very clearly documented or hinted in the interface, I obtained this knowledge by trial and error on our dev instance.

EDIT : this requires the activation of the full company support, for a reason I don't fully understand.

EDIT2 : it seems that if all users are in the same company, then all managers can see all users without consideration for the management or department fields. It is extremely furstrating
